additional-features.md

```diff
@@ -54,6 +55,62 @@ Or a `422 Unprocessably Entity` status code, with an error description in the bo
 
 Once you have the account title you can compare that with the recipient details you wish to provide us, and only create a transaction if they match.
 
+Note that an error on name enquiry might both mean that the account doesn't exist, or that there is a connectivity issue with the banking system. Because of this if you get an error message you might need to retry the call a few minutes later.
+
+## Name validation in transactions
+
+An other feature to limit mispayments because of mistyped account numbers is enabling name validation on transactions. This feature will block payouts if the account holder's name and the recipient name provided don't match. The feature is currently available for `NGN::Bank` payouts only.
```

Comment:
Typo
```suggestion
Another feature to limit mispayments because of mistyped account numbers is enabling name validation on transactions. This feature will block payouts if the account holder's name and the recipient name provided don't match. The feature is currently available for `NGN::Bank` payouts only.
```